libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -57,12 +57,14 @@ const selector = 'eo-landing-page-header';
           background: var(--watt-color-primary);
           color: var(--watt-color-primary-contrast);
 
-          // This is equivalent to a normal size watt button
+          // This is equivalent to a normal size watt button width
           min-width: 6.25rem;
-          height: 2.75rem;
+          height: calc(5 * var(--watt-space-s)); // 40px
+          padding: var(--watt-space-s) var(--watt-space-m) var(--watt-space-s)
+            var(--watt-space-m); // 8px 16px 8px 16px
 
           // Adjustment locally to fit the position of the text vertically
-          line-height: 2.75rem;
+          line-height: calc(3 * var(--watt-space-s)); // 24px;
```

Comment:
suggestion: If line height is no longer the button size, use 1.5 text size. Verify that the following works and adjust the comment above.
```suggestion
          line-height: 1.5;
```